nilearn/datasets/tests/test_neurovault.py

```diff
@@ -319,42 +319,22 @@ def test_scroll_server_results():
 def test_is_null():
     is_null = neurovault.IsNull()
     assert is_null != "a"
-    assert not is_null != ""
-    assert "a" != is_null
-    assert not "" != is_null
-    assert not is_null == "a"
     assert is_null == ""
-    assert not "a" == is_null
-    assert "" == is_null
     assert str(is_null) == "IsNull()"
```

Comment:
I get that this test and the following are aimed at testing the `__eq__` magic method of those different classes but is there any reason to test all of the following:

```python
assert is_null != "a"
assert not is_null == "a"
assert "a" != is_null
assert not "a" == is_null
```

I simplified the tests assuming that those are just unneeded test duplicates.

Let me know if I am missing something.

If there is any reason to keep those, it is then probably to use some sort of test parametrization to make the tests more readable.

Reply:
I think you're right.